datafusion/core/tests/fuzz_cases/sort_preserving_repartition_fuzz.rs

```diff
@@ -181,11 +181,14 @@ mod sp_repartition_fuzz_tests {
 
         // Fill columns based on equivalence groups
         for eq_group in eq_properties.eq_group().iter() {
-            let representative_array =
-                get_representative_arr(eq_group, &schema_vec, schema.clone())
-                    .unwrap_or_else(|| generate_random_array(n_elem, n_distinct));
+            let representative_array = get_representative_arr(
```

Comment:
I'm not at all familiar with this code, but should this be passing `eq_properties.eq_group()` directly to `get_representative_arr` instead of iterating??